ietf/doc/templatetags/ietf_filters.py

```diff
@@ -906,13 +906,17 @@ def mtime(path):
     """Returns a datetime object representing mtime given a pathlib Path object"""
     return datetime.datetime.fromtimestamp(path.stat().st_mtime).astimezone(ZoneInfo(settings.TIME_ZONE))
 
+@register.filter
+def mtime_is_epoch(path):
+    return path.stat().st_mtime == 0
+
 @register.filter
 def url_for_path(path):
     """Consructs a 'best' URL for web access to the given pathlib Path object.
 
     Assumes that the path is into the Internet-Draft archive or the proceedings.
     """
-    if path.match(f"{settings.AGENDA_PATH}/**/*"):
+    if Path(settings.AGENDA_PATH) in path.parents:
```

Comment:
These are not exactly synonymous: 
```python
>>> p = Path("/a/b/c")
>>> Path("/a/b/") in p.parents
True
>>> p.match("/a/b/**/*")
False
```

Reply:
Right - this is exactly one of the issues this PR is addressing.

Reply:
Ah- great!